<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settings</class>
 <widget class="QDialog" name="settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="languageComboBox">
     <property name="toolTip">
      <string>Select the language file to be used at application start</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>RPM Root Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="rpmRootPathEdit">
     <property name="toolTip">
      <string>Path to the rpm build enviroment</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Default Vendor:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="vendorLineEdit">
     <property name="toolTip">
      <string>Default vendor. Information used when a new project is created</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Default Packager:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="packagerLineEdit">
     <property name="toolTip">
      <string>Default packager. Information used when a new project is created</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="homepageLineEdit">
     <property name="toolTip">
      <string>Default homepage. Information used when a new project is created.</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QLineEdit" name="sourceLineEdit">
     <property name="toolTip">
      <string>Default homepage. Information used when a new project is created.</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Default Homepage:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Default Source0:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Default Buildroot:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="buildrootLineEdit">
     <property name="toolTip">
      <string>Default homepage. Information used when a new project is created.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Default Source1:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="source1LineEdit">
     <property name="toolTip">
      <string>Default homepage. Information used when a new project is created.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settings</receiver>
   <slot>slot_okButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>settings</receiver>
   <slot>slot_browseButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>471</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_okButtonClicked()</slot>
  <slot>slot_browseButtonClicked()</slot>
 </slots>
</ui>
